kindcat_composable.cue

```diff
@@ -17,12 +17,17 @@ Composable: S={
 	// Grafana and tooling with any additional schema interfaces that may be added.
 	schemaInterface: string
 	// TODO is it worth doing something like below, given that we have to keep this set open for forward compatibility?
-//	schemaInterface: or([ for k, _ in schemaInterfaces {k}, string])
+	//	schemaInterface: or([ for k, _ in schemaInterfaces {k}, string])
 
 	let schif = schemaInterfaces[S.schemaInterface]
 
 	// lineage is the Thema lineage containing all the schemas that have existed for this kind.
 	// The name of the lineage is constrained to the name of the schema interface being implemented.
+
+	if S.name != _|_ {
```

Comment:
The `name` is mandatory in the schema and this check shouldn't be necessary. 

But without that, I cannot do the tests here and make it works in Grafana at the same time. In Grafana we are setting the names by code and it generates conflicts if we remove this check.